src/test/java/com/werfen/report/FormReportTest.java

```diff
@@ -27,7 +28,21 @@ public void generateFormReport() throws IOException, JRException {
         File file = formReportService.build(this.getConfiguration(fileName + ReportFormat.PDF.getFileExtension()), this.getData(), PageFormat.A4);
         file.createNewFile();
 
-        PDDocument original = PDDocument.load(new File(fileName + GOLDEN_SUFFIX + ReportFormat.PDF.getFileExtension()));
+        PDDocument original = PDDocument.load(new File(GOLDEN_PATH + fileName + GOLDEN_SUFFIX + ReportFormat.PDF.getFileExtension()));
```

Comment:
We shall wait till Alfons merge its PR. Otherwise we will have conflicts. We need to stablish a communication channels for this. No communication means conflicts.